pkg/webhooks/sdnmigration/sdnmigration.go

```diff
@@ -46,6 +47,10 @@ type NetworkConfigWebhook struct {
 // Authorized will determine if the request is allowed
 func (w *NetworkConfigWebhook) Authorized(request admissionctl.Request) admissionctl.Response {
 	// allow if modified by an allow listed service account
+	if priviledgeUserAccount == request.UserInfo.Username {
```

Comment:
This reads backwards to me. We want to compare the request username to our constant, not the other way around.

Reply:
change done